otterize-kubernetes/values.yaml

```diff
@@ -2,6 +2,7 @@ deployment:
   spire: true
   spireIntegrationOperator: true
   intentsOperator: true
+  networkMapper: true
 
 global:
   otterizeCloud:
```

Comment:
Does the mapper chart use these credentials?

Reply:
yes